src/snmp_agent.c

```diff
@@ -109,13 +110,13 @@ static u_char snmp_agent_get_asn_type(oid *oid, size_t oid_len) {
   struct tree *node = get_tree(oid, oid_len, g_agent->tp);
 
   return (node != NULL) ? mib_to_asn_type(node->type) : 0;
-}
+} /* u_char snmp_agent_get_asn_type */
```

Comment:
I know that lot's of existing code in the code base uses these kinds of comments, but this plugin doesn't and introducing them doesn't gain much. It's easy enough to see where a function ends.